richtext-ui/src/commonMain/kotlin/com/halilibo/richtext/ui/string/Text.kt

```diff
@@ -46,9 +48,14 @@ public fun RichTextScope.Text(
       overflow = overflow,
       maxLines = maxLines,
       onClick = { offset ->
-        annotated.getStringAnnotations(Format.FormatAnnotationScope, offset, offset)
+        annotated.getStringAnnotations(RichTextString.Format.FormatAnnotationScope, offset, offset)
```

Comment:
Did this accidentally get un-imported?

Reply:
apparently. I had a problem with code styling while working on multiple machines and things got weird at some point. Thanks for catching this.